website/docs/content/writing-style/index.md

```diff
@@ -282,15 +282,15 @@ We are excited to announce Vault 1.3!
 
 ### Percentages
 
-Use the % symbol instead of spelling out "percent"; e.g. , `50%`.
+Use the % symbol instead of spelling "percent," e.g., `50%`.
```

Comment:
```suggestion
Use the % symbol instead of writing “percent,” e.g., “50%.”
```

Reply:
"50%" is just a piece of written text, not a piece of code or a value used within code, so it should not be marked with backticks.